controller/errdefs/build.go

```diff
@@ -1,48 +1,20 @@
 package errdefs
 
-import (
-	"io"
-
-	"github.com/containerd/typeurl/v2"
-	"github.com/docker/buildx/util/desktop"
-	"github.com/moby/buildkit/util/grpcerrors"
-)
-
-func init() {
-	typeurl.Register((*Build)(nil), "github.com/docker/buildx", "errdefs.Build+json")
-}
-
 type BuildError struct {
```

Comment:
This doesn't seem to be doing anything atm. As it doesn't have any extra properties and just forwards to internal error.

Reply:
Yea it's current only use is to detect a specific method of failure [here](https://github.com/docker/buildx/blob/6df3a0928478a9584e469f8527ed2c23a6f10c33/monitor/commands/reload.go#L50-L55). I think that we can fully remove this once we do a bit of a refactor to how the debug adapter works with the DAP work. For now, it was easier to keep this as a sentinel error for reload.